src/Tribe/Aggregator/Errors.php

```diff
@@ -53,7 +53,7 @@ private function __construct() {
 		tribe_register_error( 'core:aggregator:invalid-csv-parameters', __( 'Invalid data provided for CSV import.', 'the-events-calendar' ) );
 		tribe_register_error( 'core:aggregator:invalid-edit-record-type', __( 'Only scheduled import records can be edited.', 'the-events-calendar' ) );
 		tribe_register_error( 'core:aggregator:invalid-event-id', __( 'Unable to find an event with the ID of %s.', 'the-events-calendar' ) );
-		tribe_register_error( 'core:aggregator:invalid-event-json', __( 'The Event Aggregator API responded with bad data. Please <a href="https://theeventscalendar.com/support/post/" target="_blank">contact support</a>.', 'the-events-calendar' ) );
+		tribe_register_error( 'core:aggregator:invalid-event-json', sprintf( __( 'The Event Aggregator API responded with bad data. Please <a href="%s" target="_blank">contact support</a>.', 'the-events-calendar' ), 'https://theeventscalendar.com/support/post/' ) );
```

Comment:
use `%1$s` even though there's only one?

Reply:
why not use this as an opportunity to get the HTML out too?